index.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html lang="en" ng-app="ToDoList">
+  <head>
+    <meta charset="utf-8">
+    <title>To Do List</title>
+    <script src="bower_components/jquery/dist/jquery.js"></script>
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css">
+    <link href="bower_components/angular-xeditable/dist/css/xeditable.css" rel="stylesheet">
+    <link href="public/stylesheets/style.css" rel="stylesheet">
+    <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/angular-resource/angular-resource.js"></script>  
+    <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script src="bower_components/angular-xeditable/dist/js/xeditable.js"></script>
+    <script src="js/app.js"></script>
+    <script src="js/toDoListController.js"></script>
+    
+  </head>
+
+  <body ng-controller="ToDoListController as toDoCtrl">
+    <div>
+      <form class="form-horizontal" id="addTask" ng-submit="toDoCtrl.addTask()">
+        <input id='newTask' type="text" ng-model="toDoCtrl.newTask" placeholder="What Needs to be Done?">
+        <!-- <button class="btn btn-default" id="addTask" ng-click="toDoCtrl.addTask()">Add Task</button> -->
+      </form>
+      
+      <table class="table table-bordered table-hover table-condensed table-striped">
+        <tr style="font-weight: bold">
+          <td style="width:25%" class="primary">Task</td>
+          <td style="width:25%">Completed</td>
+          <td style="width:25%">Edit</td>
+        </tr>
+        <tr ng-repeat="task in toDoCtrl.taskList| filter:filterGroup | orderBy: '-completed' : 'reverse'">
+          <td> 
+            <input type="checkbox" ng-model="task.completed" aria-label="Toggle ngHide">
+            <span id='editBox' editable-text="task.description" e-name="description" e-form="rowform">
+              {{task.description || 'empty'}}
+            </span>
+          </td>
+          <td> 
+            <span class="glyphicon glyphicon-ok" ng-show="task.completed"></span>
+            <span class="glyphicon glyphicon-remove" ng-hide="task.completed"></span>
+          </td>
+          <td style="white-spave: nowrap">
+            <form editable-form name="rowform" ng-show="rowform.$visible" class="form-buttons form-inline" shown="newTask == task">
+              <button type="submit" ng-disabled="rowform.$waiting" class="btn btn-primary" id='saveEdit'>
+                save
+              </button>
+              <button type="button" ng-disabled="rowform.$waiting" ng-click="rowform.$cancel()" class="btn btn-default">
+                cancel
+              </button>
+            </form>  
+            <div class="buttons" ng-show="!rowform.$visible">
+              <button class="btn btn-primary" id='editTask' ng-click="rowform.$show()">edit</button>
```

Comment:
quite a lot of duplication. The element is already called button. Then it has 2 classes 'btn' too.